config/environments/production.rb

```diff
@@ -77,6 +77,7 @@
   #config.assets.precompile += %w( .svg .eot .woff .ttf )
   config.assets.precompile = config.assets.precompile.map!{ |path| path == /(?:\/|\\|\A)application\.(css|js)$/ ? /(?<!jquery-sortable)(?:\/|\\|\A)application\.(css|js)$/ : path }
   config.assets.precompile += %w( admin/_bootstrap-theme.min.css admin/admin.css admin.js admin-head.js )
+  config.assets.precompile += %w( congress-images-102x125/i/P000197.jpg )
```

Comment:
Is this needed?

Reply:
I think so? The tweet action feature was failing in CI with an "asset not declared to be precompiled in production" error -- see [this build](https://travis-ci.org/EFForg/action-center-platform/builds/526691906#L3506)

Reply:
Ah ok! I was unsure since it's the image for a specific rep (Pelosi). I've been testing and it looks like we'll need to precompile the whole set:

```ruby
  config.assets.precompile += %w( congress-images-102x125/i/* )
```

You can test on /action/tell-congress-to-investigate-cbp-social-media-monitoring -- when you enter a non-SF address, the image request 500s.